### DISCUSSION TITLE
Page number extracting issue

### DISCUSSION THREAD

**Discussion by Arif11112000** [pymupdf/PyMuPDF#2159] [Looking for help] [answered]
Hi Jorj, 
How are you?
I need your help.
When I extracting pdf footer page number I am getting wrong page 
numbers for certain fonts
 For example,
When I am extracting 
Footer page no 10 but getting 11

I am using below code 
import fitz
doc=fitz.open("xx.pdf")
page=doc[11]
text=page.get_text()
text=text.splitlines()
print(text[1])

OUTPUT:
Actual Output     -11
Expected Output-10

I checked manually when copied 10 from pdf footer and paste in notepad it shows 11 For your reference I attached a image .!![IMG_20230105_145830.jpg](https://user-images.githubusercontent.com/122011746/210746859-3e0fccaa-5bb5-4714-84cd-680dbc520cbd.jpg)
I don't know whether it is a font issue or not
Please help to me with this issue

**Comment by JorjMcKie** [comment, marked as answer]
Everything ok: `page=doc[11]` is page number 12 in 0-based counting scheme.
But a PDF also supports page labels, which can be literally anything the PDF creator likes to see, e.g. "A-1" or Roman numbers.
So your situation may also be blurred by this.
PyMuPDF also supports page labels, and you get see what your page's label is by `page.get_label()`:
```python
import fitz
doc=fitz.open("pymupdf.pdf")
page=doc[4]
page.get_label()
'iii'
```

**Reply by Arif11112000** [reply]
Thank You! Jorj
I got the output
